src/native/libs/System.Globalization.Native/pal_icushim_native_static.c

```diff
@@ -0,0 +1,150 @@
+#include <unicode/ucurr.h>
```

Comment:
This should not be needed. The existing `STATIC_ICU` has the same effect - just need to define it when building.

Reply:
Addressed in next revision